content/posts/2022-features.md

```diff
@@ -0,0 +1,75 @@
+---
+title: "2022 for Packit"
+date: 2023-01-28T11:58:50+01:00
+---
+
+## Packit project in 2022
+
+As you will see in the following paragraphs, the year 2022 was really fruitful for the Packit project. Without further ado, let’s take a look at what the Packit team accomplished last year:
+
+## Fedora automation
+
+Packit team has made a huge improvement in downstream automation. At the beginning of the year, we [finished the workflow](https://packit.dev/posts/downstream-automation/) and you are now able to use Packit to get your release from upstream via dist-git and Koji to Bodhi. As usual, you can pick just what you need. This workflow consists of three jobs:
+
+- propose-downstream: as a reaction to an upstream release, the source archive is saved to a lookaside cache, spec-file is updated and sent as a pull request to Fedora dist-git.
+- koji-build: as a reaction to a new dist-git commit, a new Koji build is triggered (you can specify allowed authors of a commit or merged pull request).
+- bodhi-update: as a reaction to a successfully finished Koji build, a new Bodhi update is created
```

Comment:
```suggestion
- **propose-downstream**: as a reaction to an upstream release, the source archive is saved to a lookaside cache, spec-file is updated and sent as a pull request to Fedora dist-git.
- **koji-build**: as a reaction to a new dist-git commit, a new Koji build is triggered (you can specify allowed authors of a commit or merged pull request).
- **bodhi-update**: as a reaction to a successfully finished Koji build, a new Bodhi update is created
```

for better readability

Reply:
Good point. I'll use `job-name` as at other places...